query-engine/query-engine/src/context.rs

```diff
@@ -50,7 +50,11 @@ impl PrismaContext {
             .ok_or_else(|| PrismaError::ConfigurationError("No valid data source found".into()))?;
 
         // Load executor
-        let (db_name, executor) = exec_loader::load(&data_source).await?;
+        let (mut db_name, executor) = exec_loader::load(&data_source).await?;
+
+        if let Ok(schema) = std::env::var("MSSQL_TEST_SCHEMA") {
```

Comment:
This doesn't belong here. You can put that into the exec loader for mssql as well.